web/client/plugins/DrawerMenu.jsx

```diff
@@ -15,6 +15,8 @@ const {toggleControl, setControlProperty} = require('../actions/controls');
 
 const {changeMapStyle} = require('../actions/map');
 
+const {featureClose} = require('../actions/wfsquery');
```

Comment:
We are putting querybuilder / featuregrid logic inside a generic container: it's not the right place.
We already have knowledge of querybuilder in the TOC. I don't want to mix stuff too much.